packages/manager/apps/container/src/identity-documents-modal/IdentityDocumentsModal.tsx

```diff
@@ -0,0 +1,153 @@
+    ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+
+export const IdentityDocumentsModal: FunctionComponent = () => {
+    const reketInstance = useReket();
+    const shell = useShell();
+    const navigationPlugin = shell.getPlugin('navigation');
+    const [storage, setStorage] = useLocalStorage<boolean>(kycIndiaModalLocalStorageKey);
+
+    const { t } = useTranslation('identity-documents-modal');
+    const legalInformationRef = useRef<any>(null);
+
+    const [showModal, setShowModal] = useState<boolean>(false);
+
+    const getFeatures = (): Promise<Record<string, string>> =>
+        reketInstance.get(`/feature/${kycIndiaFeature}/availability`, {
+            requestType: 'aapi',
+        });
+    const getIdentityDocumentsStatus = (): Promise<any> => reketInstance.get('/me/procedure/identity');
+
+    const { data: availability } = useQuery({
+        queryKey: ['identity-documents-availability'],
+        queryFn: getFeatures,
+        retry: 1,
+        refetchOnWindowFocus: false,
+        enabled: !storage
+    });
+
+    const { data: statusDataResponse } = useQuery({
+        queryKey: ['identity-documents-status'],
+        queryFn: getIdentityDocumentsStatus,
+        retry: 1,
+        refetchOnWindowFocus: false,
+        enabled: Boolean(availability && availability[kycIndiaFeature])
+    });
+
+    const onCancel = () => {
+        setShowModal(false);
+        setStorage(true);
+    }
+
+    const onConfirm = () => {
+        setShowModal(false);
+        setStorage(true);
+        navigationPlugin.navigateTo(
+            'dedicated',
+            `#/identity-documents`,
+        );
+    }
+
+    useEffect(() => {
+        if (statusDataResponse?.status === requiredStatusKey) {
+            setShowModal(true);
+        }
+    }, [statusDataResponse?.status])
+
+    return showModal && (
+        <OsdsModal
+            dismissible={false}
+            onOdsModalClose={onCancel}
+        >
+            <div className="my-2 text-center">
+                <OsdsText
+                    level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
+                    size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+                    color={ODS_THEME_COLOR_INTENT.text}
+                    hue={ODS_THEME_COLOR_HUE._800}
+                >{t('identity_documents_modal_title')}</OsdsText>
+            </div>
+            <div className="my-2 text-center">
+                <OsdsText
+                    level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+                    color={ODS_THEME_COLOR_INTENT.text}
+                    hue={ODS_THEME_COLOR_HUE._800}
+                >
+                    {t('identity_documents_modal_description')}
+                </OsdsText>
+            </div>
+            <div className="my-1 text-center">
+                <OsdsButton
+                    slot="actions"
+                    size={ODS_BUTTON_SIZE.sm}
+                    variant={ODS_BUTTON_VARIANT.flat}
+                    inline={false}
```

Comment:
```suggestion
```
I think if you don't put inline, it's same that inline={false}
and inline (without value) it's same inline={true}